source/agora/node/admin/AdminInterface.d

```diff
@@ -86,6 +89,39 @@ public class AdminInterface : NodeControlAPI
         this.clock = clock;
     }
 
+    ///
+    public override void postLogger (string name, bool propagate,
+        Nullable!(LogLevel) level, Nullable!bool additive,
+        Nullable!bool console, Nullable!string file) @trusted
+    {
+        import ocean.util.log.AppendConsole;
+
+        Ocean.Logger logger = name == "root" ? Ocean.Log.root :
+            Ocean.Log.lookup(name);
+
+        if (!additive.isNull())
+            logger.additive = additive.get();
+        if (!level.isNull())
+            logger.level(level.get(), propagate);
+
+        if (!console.isNull() || !file.isNull())
+        {
+            logger.clear();
+            if (!console.isNull())
```

Comment:
It would be better to take this `if` part out  of `if (!console.isNull() || !file.isNull())` part.

Reply:
I mean it like follows. It looks simpler for me. But it doesn't matter.

```
if (!console.isNull() || !file.isNull())
{
    logger.clear();
}

if (!console.isNull())
{
}

if (!file.isNull())
{
}
```

Reply:
Yeah I did that change